torch/nn/functional.py

```diff
@@ -191,6 +192,10 @@ def conv_tbc(input, weight, bias, pad=0):
     return input.conv_tbc(weight, bias, pad)
 
 
+# Interface for the gradients of various operators
+grad = G
```

Comment:
If a user wants a quick access to a module by aliasing it with such a short name then fine, but we shouldn't do it for them. Let's remove this line